docker_templates/templates/snippet/install_ros_bootstrap_tools.Dockerfile.em

```diff
@@ -1,20 +1,25 @@
 @{
-if int(ros_version) == 2:
-    package_list = [
-        'build-essential',
+import rosdistro
+index = rosdistro.get_index(rosdistro.get_index_url())
+dist_info = index.distributions[ros_distro]
+ros_version = int(dist_info['distribution_type'][-1])
```

Comment:
I'm guessing this is parsing out the number from a string like `v1` or `v2`?

Reply:
Indeed, I can add a comment to be more explicit.
This s based on the [REP defining distribution index yaml](https://www.ros.org/reps/rep-0153.html#index-file) structure. Possible values for ros distributions are `ros1` and `ros2` so here we extract the number 1 or 2

Reply:
Added comment here https://github.com/osrf/docker_templates/pull/103/commits/99b4fd403f170610c289c5f1baa3712d06c65cf0